Classes/Notifications/NotificationsViewController.swift

```diff
@@ -184,9 +261,10 @@ FlatCacheListener {
     func models(listAdapter: ListAdapter) -> [ListDiffable] {
         var models = [NotificationViewModel]()
 
+        let showAll = self.showAll
```

Comment:
Why shadow?

Reply:
Don't need to recalc every time in the loop. Grab it once locally and use that instead.

Reply:
Ah yep, didn't realise it's computed. 👍